integreat_cms/cms/fixtures/test_data.json

```diff
@@ -3566,6 +3566,18 @@
       "ignore": false
     }
   },
+  {
+    "model": "linkcheck.link",
+    "pk": 12,
+    "fields": {
+      "content_type": 23,
+      "object_id": 61,
+      "field": "content",
+      "url": 3,
+      "text": "Gültiger Link",
```

Comment:
```suggestion
      "text": "Ein langer Link zum Testen des Verhaltens von Links im PDF Export. Text mit Leerzeichen sollte gebrochen werden, Wörter mit mehr als 50 Zeichen sollten hingegen gekürzt werden. www.dashieristeinsehrlangerlinkalszusammenhängendezeichenkette.de",
```
When I import the test data and run `./tools/integreat-cms-cli findlinks`, I get the following error:
```
linkcheck.models.Link.MultipleObjectsReturned: get() returned more than one Link -- it returned 2!
```

So apparently it's not possible to have the same combination of link text and url twice.

Reply:
Oh yes, I missed that 🙈